src/utils/decorators/tooltip-decorator/tooltip-decorator.js

```diff
@@ -339,7 +339,7 @@ const TooltipDecorator = (ComposedComponent) => {
     get tooltipHTML() {
       if (!this.props.tooltipMessage) { return null; }
       return (
-        <Portal open={ this.state.isVisible }>
+        this.state.isVisible && <Portal>
```

Comment:
maybe this could be combined with line 340, something like

```js
if (!this.props.tooltipMessage || !this.state.isVisible) return null;
```

Reply:
How about ?
```js
(this.props.tooltipMessage && this.state.isVisible)  && <Portal>
```